dir.props

```diff
@@ -49,6 +46,15 @@
     <ProjectPackagesConfigFile>$(NuGetConfigFile)</ProjectPackagesConfigFile>
   </PropertyGroup>
   
+  <!-- Copy the "_._" file to the output directory, which is used by the NuSpecs to indicate an empty folder -->
+  <ItemGroup>
+    <None Include="$(SourceDir)\nuget\_._">
+      <Link>_._</Link>
```

Comment:
I think it would be good to comment somewhere why there needs to be a file called _._ :smile: